lib/screens/activity_list_view.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:maui/components/topic_button.dart';
+import 'package:maui/db/entity/activity.dart';
+import 'package:maui/loca.dart';
+import '../repos/activity_repo.dart';
+
+class ActivityListView extends StatefulWidget {
+  const ActivityListView({Key key}) : super(key: key);
```

Comment:
Let this take a topicId parameter and for now, you can assign default value of 'lion'